releasetasks/templates/beetmover_enUS_push_to_candidates.yml.tmpl

```diff
@@ -0,0 +1,57 @@
+{% for platform, platform_info in en_US_config["platforms"].iteritems() %}
+{% set buildername = "release-{}_{}_{}_en-US_beetmover_candidates".format(branch, product, platform) %}
+-
+    taskId: "{{ stableSlugId(buildername) }}"
+    reruns: 5
+    task:
+        provisionerId: aws-provisioner-v1
+        workerType: opt-linux64
+        created: "{{ now }}"
+        deadline: "{{ now.replace(hours=6) }}"
+        expires: "{{ never }}"
+        priority: "high"
+        retries: 5
+        payload:
+            maxRunTime: 7200
+            # TODO - create specific image for this
+            image: rail/python-test-runner
+            command:
+                - /bin/bash
+                - -c
+                - >
+                  wget -O mozharness.tar.bz2 https://hg.mozilla.org/releases/{{ branch }}/archive/{{ revision }}.tar.bz2/testing/mozharness &&
+                  mkdir mozharness && tar xvfj mozharness.tar.bz2 -C mozharness --strip-components 3 &&
+                  python scripts/release/beet_mover.py --template configs/beetmover/en_us.yml.tmpl --platform {{ platform_info['upload_platform'] }} --version {{ version }} --app-version {{ appVersion }} --locale en-US --taskid {{ platform_info['task_id'] }} --build-num build{{ buildNumber }}
+            encryptedEnv:
+                - {{ encrypt_env_var(stableSlugId(buildername), now_ms,
+                                   now_ms + 24 * 3600 * 1000, 'AWS_ACCESS_KEY_ID',
+                                   beetmover_aws_access_key_id) }}
+                - {{ encrypt_env_var(stableSlugId(buildername), now_ms,
+                                   now_ms + 24 * 3600 * 1000, 'AWS_SECRET_ACCESS_KEY',
+                                   beetmover_aws_secret_access_key) }}
+        metadata:
+            name: "{{ product }} {{ branch }} {{ platform }} en_US beetmover"
+            description: "Release Promotion en_US beetmover job"
+            owner: "release@mozilla.com"
+            source: https://github.com/mozilla/releasetasks
+
+        {% if running_tests is defined %}
+        extra:
+            task_name: "{{ buildername }}"
+            signing:
+               signature: {{ sign_task(stableSlugId(buildername), valid_for=4 * 24 * 3600) }}
```

Comment:
The "signing" block should be outside of the "if" condition, so we sign the tasks in production too. I wonder how to add tests to avoid this in the future...